example-crates/wallet_esplora_blocking/src/main.rs

```diff
@@ -1,19 +1,22 @@
-const DB_MAGIC: &str = "bdk_wallet_esplora_example";
-const SEND_AMOUNT: Amount = Amount::from_sat(1000);
-const STOP_GAP: usize = 5;
-const PARALLEL_REQUESTS: usize = 1;
+use std::{io::Write, str::FromStr};
 
-use std::{collections::BTreeSet, io::Write, str::FromStr};
+use tempfile::tempdir;
 
 use bdk_esplora::{esplora_client, EsploraExt};
 use bdk_file_store::Store;
 use bdk_wallet::{
-    bitcoin::{Address, Amount, Network},
+    bitcoin::{Address, Amount, Network, Script},
     KeychainKind, SignOptions, Wallet,
 };
 
+const DB_MAGIC: &str = "bdk_wallet_esplora_blocking_example";
+const SEND_AMOUNT: Amount = Amount::from_sat(5000);
+const STOP_GAP: usize = 50;
```

Comment:
50 seems like a lot for these examples

Reply:
Done in 214fa09d368851b352b827b31608a986600702ec

Changed to 20